exist-parent/pom.xml

```diff
@@ -1228,13 +1239,24 @@
                 <enabled>true</enabled>
             </snapshots>
         </pluginRepository>
+        <pluginRepository>
```

Comment:
This is not needed as there are no plugins that we publish.

Reply:
@duncdrum was this needed for snapshots of the sub-projects to be published to the registry?